libr/util/str.c

```diff
@@ -2357,8 +2357,14 @@ R_API bool r_str_glob(const char* str, const char *glob) {
 			if (!*++glob) {
 				return true;
 			}
+			if (*glob == '*') {
```

Comment:
you can join both conditionals with && , but for readability maybe better to have `glob++;\nif (!glob) return true;}`, as separate lines. not blocking just commenting in case you think its better one way or the other.